src/_vars.scss

```diff
@@ -7,6 +7,7 @@
 
 // For use in a .js file, see `src/theme.js`.
 
+// DEPRECATED by theme palette update, colors here should be removed eventually in the future!
```

Comment:
As we discussed in person, every color needs a (single) name.  That name should be used in the theme definitions.  If you want to delete this name eventually, then create a new better name and use that.  (In fact, use the good name to define the old name, if possible).   In general, each hex code should only appear once anywhere.

Reply:
As we discussed, the _vars file will be deprecated by this update, and is only still here to ensure backwards compatibility while we transition.  Because of that, there's no need that I can see to updated the colors here with a different naming scheme.